SpatialGDK/Source/Public/Interop/Connection/SpatialWorkerConnection.h

```diff
@@ -36,6 +36,9 @@ class SPATIALGDK_API USpatialWorkerConnection : public UObject
 
 	OnConnectedDelegate OnConnected;
 
+	FReceptionistConfig ReceptionistConfig;
```

Comment:
Could we use accessors so that is easier to track where they are changed?

Reply:
Discussed offline, leaving as public for now.